Resources/Prototypes/_RMC14/Entities/Structures/Machines/Vending/Crew/auxiliary.yml

```
@@ -0,0 +1,234 @@
+# Flight Crew Vendors #
+
+# Equipment Rack
+- type: entity
+  parent: ColMarTechBase
+  id: ColMarTechDropshipCrewEquipment
+  name: ColMarTech Dropship Crew Equipment Rack
+  description: An automated equipment vendor for Dropship Crew
+  components:
+  - type: AccessReader
+    access:
+    - [ "CMAccessDropship" ]
```

This needs to be pilot, not dropship, dropship is the ability to open the doors in general which all marines have
```suggestion
    - [ "CMAccessPilot" ]
```